src/service/mobile_config.proto

```diff
@@ -21,15 +21,15 @@ message gateway_metadata {
   // The res12 h3 index asserted address of the gateway as a string
   // where an unasserted gateway returns an empty string
   string location = 2;
-  // the asserted device type of the gateway
-  string device_type = 3;
 }
 
 message gateway_info {
   // The public key binary address and on-chain identity of the gateway
   bytes address = 1;
   // The gateway metadata as recorded on the blockchain
   gateway_metadata metadata = 2;
+  // the asserted device type of the gateway
+  device_type device_type = 3;
```

Comment:
why did you move this out of gateway_metadata?

Reply:
Internally metadata is an Option which makes accessing fields a lil more cumbersome but you are right in that the device_type is onchain as part of the metadata. I can move it back, no big deal

Reply:
After discussing we decided to revert/keep the device type in the top-level GatewayInfo. Key reasons were:
- it's similar to `is_full_hotspot` in that it should be an intrinsic aspect of the hotspot instead of a fungible element like asserted location
- it makes the handling of the data easier with fewer nested "options"